### ISSUE TITLE
Should pathogen_summary and list_distributions be merged into a single function?

### ISSUE THREAD

**Issue by joshwlambert** [epiverse-trace/epiparameter#48]
Both `list_distributions()` and `pathogen_summary()` both read in the parameters data, filter the data and return a data frame. The main difference between the functions is `pathogen_summary()` calculates the mean and standard deviation. It might be simpler to reduce them into a single function.

**Comment by adamkucharski** [comment]
Agree, although there may be benefit having an object (like `list_distributions()`) returned that has sufficiently few columns to see the list clearly? But this could still be done in single function.
